lib/Common/Memory/SmallFinalizableHeapBucket.cpp

```diff
@@ -93,15 +93,6 @@ SmallFinalizableHeapBucketBaseT<TBlockType>::AggregateBucketStats(HeapBucketStat
 }
 #endif
 
-template void SmallFinalizableHeapBucketBaseT<SmallFinalizableHeapBlock>::Sweep(RecyclerSweep& recyclerSweep);
-template void SmallFinalizableHeapBucketBaseT<MediumFinalizableHeapBlock>::Sweep(RecyclerSweep& recyclerSweep);
-
-#ifdef RECYCLER_WRITE_BARRIER
-template void SmallFinalizableHeapBucketBaseT<SmallFinalizableWithBarrierHeapBlock>::Sweep(RecyclerSweep& recyclerSweep);
-template void SmallFinalizableHeapBucketBaseT<MediumFinalizableWithBarrierHeapBlock>::Sweep(RecyclerSweep& recyclerSweep);
-#endif
```

Comment:
These no longer need to be explicitly instantiated because we instantiate the class anyway. They used to be explicitly instantiated before because sweep was itself templatized but that is also no longer the case.